frontend/src/components/opportunity/OpportunityDescription.tsx

```diff
@@ -76,13 +81,15 @@ const OpportunityDescription = ({ opportunityData }: Props) => {
         <div
           dangerouslySetInnerHTML={{
             __html: DOMPurify.sanitize(
-              opportunityData.summary.agency_contact_description,
+              opportunityData.summary.agency_contact_description
+                ? opportunityData.summary.agency_contact_description
+                : "",
             ),
           }}
         />
         <p>{opportunityData.summary.agency_email_address}</p>
```

Comment:
Should this also be changed to `{agency_email}`

Reply:
Updated, thanks.